plugins/http/src/commands.rs

```diff
@@ -246,7 +224,13 @@ pub async fn fetch<R: Runtime>(
                     request = request.header(header::USER_AGENT, "tauri-plugin-http");
                 }
 
-                request = request.header(header::ORIGIN, webview.url()?.as_str());
+                if !(cfg!(feature = "unsafe-header")
+                    && headers.contains_key(header::ORIGIN.as_str()))
+                {
+                    if let Ok(url) = webview.url() {
+                        request = request.header(header::ORIGIN, url.as_str());
```

Comment:
we're using the full url, shouldn't we only use scheme://hostname:port?

Reply:
that's right, I didn't know that tbh

Reply:
should be fixed now